v2/guides/blocks/architecture-with-sdk.mdx

```diff
@@ -0,0 +1,29 @@
+
+import { GuideRecipeLink } from "/src/components/guides/GuideRecipeLink"
```

Comment:
not related to this file, but we also need in the backend section, a generic "Other frameworks" option which can talk about the other way of integrating supertokens. We can assume in there the use of .NET, but say to the user that the guide is pretty much the same even for other application backends